internal/config/profile.go

```diff
@@ -241,9 +241,23 @@ func (p *Profile) Name() string {
 	return p.name
 }
 
-func SetName(name string) { Default().SetName(name) }
-func (p *Profile) SetName(name string) {
+func validateName(name string) error {
+	if strings.Contains(name, ".") {
+		return errors.New("Profile should not contain '.'")
+	}
+
+	return nil
+}
```

Comment:
let's make this a bit friendlier
```suggestion
var ErrProfileNameHasDots = errors.New("profile should not contain '.'")
func validateName(name string) error {
	if strings.Contains(name, ".") {
		return fmt.Errorf("%w: %q", ErrProfileNameHasDots, name)
	}

	return nil
}
```